ihmc-communication/src/main/java/us/ihmc/communication/ROS2Tools.java

```diff
@@ -109,6 +112,31 @@ public static ROS2Node createLoopbackROS2Node(PubSubImplementation pubSubImpleme
       return new ROS2Node(pubSubImplementation, nodeName, FACTORY.getDomainId(), loopbackAddress);
    }
 
+   /**
+    * Creates a ROS2 node that only communicates using Fast-DDS shared memory (SHM) transport.
+    * See: <a href="https://fast-dds.docs.eprosima.com/en/v2.14.3/fastdds/transport/shared_memory/shared_memory.html">Shared Memory Transport</a>
+    * This is useful when publishing large messages for intra-process or local-only communication, as to not overwhelm the network.
```

Comment:
This applies to _inter_process comms on the same machine, right?

Reply:
Yes